modules/osd-storage-pv-aws-uninstall-efs.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
+
+[id="osd-storage-pv-aws-uninstall-efs_{context}"]
+= Uninstalling the EFS Operator
+
+.Procedure
+
+To remove the operator from your cluster:
+
+. Delete all of the workloads using the PersistentVolumeClaims that were generated by the operator.
+. Delete all of the SharedVolumes from all of the namespaces. The operator automatically removes the associated PersistentVolumes and PersistentVolumeClaims.
```

Comment:
s/SharedVolumes/\`SharedVolume` resources

s/operator/Operator

s/PersistentVolumes and PersistentVolumeClaims/\`PersistentVolumes\`and \`PersistentVolumeClaim` objects.